__tests__/news/news.mutation.test.js

```diff
@@ -0,0 +1,260 @@
+/* eslint-disable object-curly-newline */
+/* eslint-disable no-undef */
+const { gql } = require('apollo-boost');
+const client = require('../../utils/apollo-test-client');
+
+let NEWS_ID = '';
+const NEWS_DOES_NOT_EXIST_ID = '1f2ad470eb08183384e6111b';
+describe('test news mutations', () => {
+  test('#1 should add news to database', async () => {
+    const res = await client
+      .mutate({
+        mutation: gql`
+          mutation {
+            addNews(
+              news: {
+                title: [
+                  { lang: "uk", value: "bbb" }
+                  { lang: "eng", value: "bbb" }
+                ]
+                text: [
+                  { lang: "uk", value: " d a s d" }
+                  { lang: "eng", value: " a s d" }
+                ]
+                author: {
+                  name: [
+                    { lang: "uk", value: "a sd" }
+                    { lang: "eng", value: "a sd" }
+                  ]
+                }
+                images: {
+                  primary: { medium: "ada s.jpg" }
+                  additional: [{ medium: "as dasdsa.jpg" }]
+                }
+                date: "1111118820047"
+              }
+            ) {
+              ... on News {
+                _id
+                title {
+                  lang
+                  value
+                }
+                text {
+                  lang
+                  value
+                }
+                author {
+                  name {
+                    lang
+                    value
+                  }
+                }
+              }
+              ... on Error {
+                message
+                statusCode
+              }
+            }
+          }
+        `,
+      })
+      .then(res => res)
+      .catch(e => e);
+    NEWS_ID = res.data.addNews._id;
+
+    expect(res.data.addNews.title).toBeInstanceOf(Array);
+    expect(res.data.addNews).toHaveProperty('title', [
+      { __typename: 'Language', lang: 'uk', value: 'bbb' },
+      { __typename: 'Language', lang: 'eng', value: 'bbb' },
+    ]);
+    expect(res.data.addNews.text).toBeInstanceOf(Array);
+    expect(res.data.addNews).toHaveProperty('text', [
+      { __typename: 'Language', lang: 'uk', value: ' d a s d' },
+      { __typename: 'Language', lang: 'eng', value: ' a s d' },
+    ]);
+    expect(res.data.addNews.author).toBeInstanceOf(Object);
+    expect(res.data.addNews).toHaveProperty('author', {
+      __typename: 'Author',
+      name: [
+        { __typename: 'Language', lang: 'uk', value: 'a sd' },
+        { __typename: 'Language', lang: 'eng', value: 'a sd' },
+      ],
+    });
+  });
+  test('#2 update news', async () => {
```

Comment:
Also, please, add the test that tries to update an article with duplicated name and gets and error. Please, check that proper error message and status code are returned

Reply:
added